File: track-web/src/models/Property.ts
export class Property {
  public Id: number;
  public RecordId: number;
  public SeriesId: number;
  public Label: string;
  public Value: string;

  constructor(seriesId: number, value: string = null) {
    this.SeriesId = seriesId;
    if (value != null) this.Value = value;
  }
}

File: track-api/Models/Db/Property.cs
namespace track_api.Models.Db
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Property")]
    public partial class Property : DbEntity
    {
        public int RecordId { get; set; }

        public int SeriesId { get; set; }

        [Required]
        public string Value { get; set; }
    }
}
